src/components/correct-name/CorrectNameToNumber.vue

```diff
@@ -4,17 +4,13 @@
     ref="form"
     v-model="formValid"
   >
-    <v-row no-gutters>
-      <v-col>
```

Comment:
A single row with a single column is a lazy way to lay out a flex panel, and in fact it's not needed here.

![image](https://github.com/bcgov/bcrs-shared-components/assets/10002889/0b443957-7951-4fa6-8f5b-ec10c1c0f156)